test/feature/rule.js

```
@@ -57,12 +57,16 @@ describe('Rule', function() {
         });
 
         it('simple value match', function() {
-            var r = new Rule('rule', {
-                topic: 'nono',
-                exec: {uri: 'a/b/c'},
-                match: {number: 1, string: 'oolala'}
-            });
-            assert.ok(r.test(msg), 'Expected the rule to match the given message!');
+            try {
```

Why? If it fails, it should fail loudly